doc/start/quick-ceph-deploy.rst

```diff
@@ -139,7 +141,9 @@ configuration details, perform the following steps using ``ceph-deploy``.
       ``--data`` *must* be ``volume_group/lv_name``, rather than the path to
       the volume's block device.
 
-#. Check your cluster's health. ::
+#. Check your cluster's health::
+
+   Log into any one of your *non-admin* nodes and execute:
```

Comment:
this line is not right, it will be formatted in monospace, as if it was part of the terminal output, it needs to be out of the `::` block (or before it)